app/actions/nodeStorageActions.js

```diff
@@ -47,6 +49,7 @@ export const getRPCEndpoint = async (
   net: Net,
   excludeCritera: Array<string> = NODE_EXLUSION_CRITERIA,
 ) => {
+  settings.timeout.ping = RPC_PING_OVERRIDE
```

Comment:
Overriding a global setting should better be avoided as it introduces the problematic "side-effect" anti-pattern. Suggestions:
1) Restore original setting on completion.
```js
const originalPing = settings.timeout.ping
settings.timeout.ping = RPC_PING_OVERRIDE // override
// do stuff
settings.timeout.ping = originalPing // restore
```
2) Update `neon-js` api to accept an override in current scope. Something like:
```js
data.client.ping(RPC_PING_OVERRIDE)
```
or
```js
const settings = {timeout: { ping: RPC_PING_OVERRIDE } }
const client = new rpc.RPCClient(url, settings)
```